geniza/footnotes/models.py

```diff
@@ -410,6 +411,43 @@ def editions(self):
         return self.filter(doc_relation__contains=Footnote.EDITION)
 
 
+class HTMLLineNumberParser(HTMLParser):
```

Comment:
not sure this belongs in models, could we put it in a utils file or something like that?